src/component/base/DraftEditor.react.js

```diff
@@ -190,6 +191,10 @@ class DraftEditor extends React.Component {
       if (!this.props.readOnly) {
         const method = this._handler && this._handler[eventName];
         method && method(this, e);
+      } else if (eventName === 'onCopy') {
+        editOnSelect(this)
```

Comment:
Talked about this with Max after trying to not edit draftjs and after exporting methods out of draft and playing with the editor, the work was getting overly complicated so we wanted to see if there was an easy to do the copy in draft since it already ALMOST does everything be default.

Happy to add a comment to about why we need to do this.